stanford/roles/cluster/tasks/emr.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Lookup/create emr slave security group
+  ec2_group:
+    # Tags added in Ansible 2.4
+    name: "{{ CLUSTER_SECURITY_GROUP }}-slave"
+    description: "{{ CLUSTER_SECURITY_GROUP_DESCRIPTION }}-slave"
+    vpc_id: "{{ DEPLOYMENT_VPC_ID }}"
+    region: "{{ CLUSTER_REGION }}"
+    rules: 
+    - proto: icmp
+      from_port: 0
+      to_port: 65535
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+    - proto: tcp
+      from_port: 0
+      to_port: 65535
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+    - proto: udp
+      from_port: 0
+      to_port: 65535
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+  register: cluster_security_group-slave
```

Comment:
Should this hyphen actually be an underscore?

Reply:
Isn't it discretionary?